### PR TITLE
form update

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: angular-techDraft/src/app/login/login.component.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+
```
Comment (JaredHook) [inline angular-techDraft/src/app/login/login.component.css:1]: extra lines

File: angular-techDraft/src/app/login/login.component.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (JaredHook) [inline angular-techDraft/src/app/login/login.component.html:1]: unnecessary lines

File: angular-techDraft/src/app/login/login.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (JaredHook) [inline angular-techDraft/src/app/login/login.component.spec.ts:2]: extra line

File: angular-techDraft/src/app/login/login.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
```
Comment (JaredHook) [inline angular-techDraft/src/app/login/login.component.ts:14]: extra line

**Review by jp-2019 — APPROVED** [review]
Excellent!

**Comment by jp-2019** [comment]
Excellent

**Comment by jp-2019** [comment]
Nice

**Review by shakirullah7892 — APPROVED** [review]

**Pull request merged**